scripts/helpers/iterators.py

```diff
@@ -0,0 +1,19 @@
+import itertools
+
+
+def chunked_iterable(iterable, *, size):
```

Comment:
This function is everywhere!

Reply:
It's so handy! 😜 

Which is kinda the point of starting to library-ise some of this stuff. These are utilities so often and rely upon, I want to spend less time copying them around, and more time trusting they work.

Reply:
They should put this one in core Python 😉